src/components/Card/Card.js

```diff
@@ -1,12 +1,18 @@
-import React from 'react'
-import './card.css'
+import React from "react";
+import "./card.css";
+import PlaceHolderImage from "../../assets/placeholder_image.png";
+
+const Card = ({ id, fullName, title, photo }) => {
```

Comment:
I think we should also receive description as props as well. Better to show the some of the description rather than a button of more info.